integrations/vero/lib/index.js

```diff
@@ -108,6 +108,29 @@ Vero.prototype.track = function(track) {
   if (tags) this.addOrRemoveTags(tags);
 };
 
+/**
+ * Order Completed.
+ *
+ * https://www.getvero.com/api/http/#actions
+ * https://github.com/getvero/vero-api/blob/master/sections/js.md#tracking-events
+ *
+ * @api public
+ * @param {Track} track
+ */
+
+Vero.prototype.orderCompleted = function(track) {
+  var products = track.properties().products;
```

Comment:
I think I had an occasional issue with `properties()` being undefined over on the client side FWIW.

Reply:
Should be ok - `properties` defaults to `{}` if otherwise undefined: https://github.com/segmentio/facade/blob/master/lib/track.js#L460-L472

Should also be able to use `track.products()` for a shortcut, but then would have to check the length of the returned array: https://github.com/segmentio/facade/blob/master/lib/track.js#L382-L386